gooddata-sdk/gooddata_sdk/catalog/workspace/content_service.py

```diff
@@ -552,7 +553,9 @@ def compute_valid_objects(self, workspace_id: str, ctx: ValidObjectsInputType) -
 
         return by_type
 
-    def get_label_elements(self, workspace_id: str, label_id: LabelElementsInputType) -> List[str]:
+    def get_label_elements(
+        self, workspace_id: str, label_id: LabelElementsInputType, depends_on: Optional[DependsOn] = None
+    ) -> List[str]:
```

Comment:
We do not want users to use classes from `gooddata_api_client` directly therefore, we create a wrapper class on top of it.